src/middlewared/middlewared/plugins/kerberos.py

```diff
@@ -672,11 +777,10 @@ async def do_create(self, data):
 
     @accepts(
         Int('id', required=True),
-        Dict(
+        Patch(
+            'kerberos_keytab_create',
             'kerberos_keytab_update',
-            Str('file'),
-            Str('name'),
-            register=True
+            ('attr', {'update': True})
```

Comment:
The `accepts` decorator on `do_update` is not required for this specific instance and it would be automatically managed by middleware ( just letting you know :) )